### PR TITLE
add month to booked (inventory array) when I seed my db

### PR DESCRIPTION
**Pull request by hannahmanfredi**

### CODE REVIEW

File: client/dist/index.html [hunk 1]
```diff
@@ -1,14 +1,59 @@
...
+    [booking]/[webpack]/lib/Compilation.js:2526:11
+  
+  
+  - Hook.js:18 Hook.CALL_ASYNC_DELEGATE [as _callAsync]
+    [booking]/[tapable]/lib/Hook.js:18:14
+  
+  - Compilation.js:2519 hooks.processAssets.callAsync.err
+    [booking]/[webpack]/lib/Compilation.js:2519:38
+  
+</pre>
```
Comment (kimberlyjk) [inline client/dist/index.html:59]: This is confusing... but I don't think you wrote this! Lol. More things to .gitignore 😅

File: database/seed.js [hunk 2]
```diff
@@ -156,7 +157,7 @@ const seedDb = async () => {
       newBooking = {
         campId: 0,
         price_per_night: 165,
-        booked: [[1, 2, 3], [15, 16, 17], [25, 26, 27], [28, 29]],
+        booked: [[6], [1, 2, 3], [15, 16, 17], [25, 26, 27], [28, 29]],
```
Comment (kimberlyjk) [inline database/seed.js:160]: What does this nested array represent?

**Review by kimberlyjk — APPROVED** [review]
I also didn't run this code. Some of the changes are slightly confusing but I think I just don't understand 100% what your seeding script is doing. 

It looks good from what I can see. Great work!

File: client/dist/index.html (hunk 1 above)
Comment (hannahmanfredi) [reply client/dist/index.html:59]: I just got my  .gitignore working. Sorry yea that weird html file is generated when my .gitignore doesn't ignore my dist folder. Figured out that I have to first run git rm -r --cached dist to remove from my git cache (aka index)

File: database/seed.js (hunk 2 above)
Comment (hannahmanfredi) [reply database/seed.js:160]: it is the month that the booked dates fall under

**Pull request closed without merging**